ONSdigital/eq-survey-runner
# Eq 374 serialiser

### Changes

Adds the MCI Schema as python code, along with to_json() methods to generate json from model.
Also overrides **eq** and **ne** methods to check for equivalence rather than strict equality.
### How to test

1) Check out the branch
2) Run the unit tests
3) Verify they all pass
### Who can test

Anyone except @weapdiv-david
